templater/jobs/presubmit/eks-anywhere-prow-jobs/eks-anywhere-prowjobs-lint-presubmits.yaml

```diff
@@ -0,0 +1,8 @@
+jobName: eks-anywhere-prowjobs-lint-presubmit
+runIfChanged: EKS_DISTRO_BASE_TAG_FILE|jobs/aws/.*
+commands:
+- make
```

Comment:
Let's modify this job to first re-generate Prowjobs using the templater and then run linter on those jobs, so that we can also verify that the jobs folder and templater folder are in sync. Similar to [this](https://github.com/aws/eks-distro-prow-jobs/blob/main/jobs/aws/eks-distro-prow-jobs/prowjobs-lint-presubmits.yaml#L44-L46).